airbyte-ci/connectors/ci_credentials/README.md

```diff
@@ -27,23 +28,22 @@ python -m pip install --user pipx
 python -m pipx ensurepath
 ```
 
-Once pyenv and pipx is installed then run the following:
+Once pyenv and pipx is installed then run the following (assuming you're in Airbyte repo root):
 
 ```bash
 pipx install --editable --force --python=python3.10 airbyte-ci/connectors/ci_credentials/
 ```
 
 This command installs `ci_credentials` and makes it globally available in your terminal.
 
-_Note: `--force` is required to ensure updates are applied on subsequent installs._
-_Note: `--python=python3.10` is required to ensure the correct python version is used._
-_Note: `--editable` is required to ensure the correct python version is used._
-
-If you face any installation problem feel free to reach out the Airbyte Connectors Operations team.
-
+> [!Note]
+> - `--force` is required to ensure updates are applied on subsequent installs.
+> - `--python=python3.10` is required to ensure the correct python version is used.
+> - `--editable` is required to ensure the correct python version is used.
```

Comment:
`- `--editable` is required to ensure the correct python version is used.`

This sounds weird to me. Editable ensures that changes to the package code do not require a re-install to be captured, it should not have anything to do with the python version.